torch/onnx/utils.py

```diff
@@ -207,15 +219,13 @@ def _decide_keep_init_as_input(keep_initializers_as_inputs, operator_export_type
         val_keep_init_as_ip = False
     return val_keep_init_as_ip
 
+
 def _decide_add_node_names(add_node_names, operator_export_type):
```

Comment:
I am wondering whether _decide_add_node_names and _decide_constant_folding can be just inlined. I will land it as it is now, but feel free to create another PR to fix it.